docs/cmdline_migration.md

```diff
@@ -0,0 +1,73 @@
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Migrating to OpenJ9
+
+If you are already familiar with HotSpot command-line options but want the advantages of OpenJ9, the following information will prove helpful.In all cases, check individual topics for minor discrepancies in the way these options might work.
+
+- You can use the following command-line options in OpenJ9, just as you did in Hotspot; you can continue to use the HotSpot option in OpenJ9 without having to change your code:
+
+| Option                                                | Usage                                                                                      |
+|-------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| [`-X`](x.md)                                          | Displays help on nonstandard options.                                                      |
+| [`-Xbootclasspath`](xbootclasspath.md)                | Specifies the search path for bootstrap classes and resources.                             |
+| [`-Xcheck:jni`](xcheck.md)                            | Runs additional checks for JNI functions during VM startup.                                |
+| [`-Xfuture`](xfuture.md)                              | Turns on strict class-file format checks.                                                  |
+| [`-Xint`](xint.md)                                    | Runs an application in interpreted-only mode.                                              |
+| [`-Xmn`](xmn.md)                                      | Sets the initial and maximum size of the new area when using -Xgcpolicy:gencon.            |
+| [`-Xms`](xms.md)                                      | Sets the initial size of the heap.                                                         |
+| [`-Xmx`](xmn.md)                                      | Specifies the maximum size of the memory allocation pool. (Equivalent to -XX:MaxHeapSize.) |
```

Comment:
The link is to the wrong topic.

This sets the maximum size of the object heap. "memory allocation pool" seems more generic. I guess "object memory allocation pool" would work.

Please clarify that OpenJ9 does not recognize -XX:ThreadStackSize

Reply:
Sorry... getting my -Xm's mixed up :)